OmiseSDK/CardNameTextField.swift

```diff
@@ -7,7 +7,7 @@ import UIKit
 public class CardNameTextField: OmiseTextField {
     /// Boolean indicating wether current input is valid or not.
     public override var isValid: Bool {
-        return !text.isNilOrEmpty
+        return !text.isNilOrEmpty && (text ?? "").rangeOfCharacter(from: .decimalDigits) == nil
```

Comment:
Doesn't `!text.isNilOrEmpty` mean that text can't be empty at this point `(text ?? "")`?

Reply:
Yep. First, we check text in not nil or empty. but for `(text ?? "")` i use shorthand nil-coalescing operator for unwraps an optional `text` if it contains a value and calls `rangeOfCharacter` method for find digits in name string.